### PR TITLE
[CHI-381] Improved behaviour of new version alert banner of header

### PR DESCRIPTION
**Pull request by janibolkvadze**
@jllr @mattnickles @dani-cl-madrid @SergioQCL 

https://ctl.atlassian.net/browse/CHI-381

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/293/ for more details.

File: src/website/assets/scripts/chiVersionCheck.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+function fillDropdown() {
+  var drop = document.getElementById('versionDropdown');
+  
+  while (drop.childNodes.length > 0) {
+    drop.removeChild(drop.firstChild);
+  }
+
+  for (let version in versions) {
```
Comment (dani-cl-madrid) [inline src/website/assets/scripts/chiVersionCheck.js]: Please don't use ES6 code in this files (let)

File: src/website/assets/scripts/chiVersionCheck.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+var versions;
+
+if (window.location.hostname === 'assets.ctl.io') {
+  var xmlhttp = new XMLHttpRequest();
+
+  xmlhttp.onreadystatechange = function() {
+    if (xmlhttp.readyState == XMLHttpRequest.DONE) {
```
Comment (dani-cl-madrid) [inline src/website/assets/scripts/chiVersionCheck.js]: Please use === instead of ==

Comment (janibolkvadze) [reply src/website/assets/scripts/chiVersionCheck.js]: All done, thanks

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/309/ for more details.

**Pull request merged**